imagehash.py

```diff
@@ -390,10 +390,23 @@ def __eq__(self, other):
 			return False
 		return self.matches(other)
 
+	def __ne__(self, other):
+		if other is None:
+			return False
+		return not self.matches(other)
+
 	def __sub__(self, other, hamming_cutoff=None, bit_error_rate=None):
 		matches, sum_distance = self.hash_diff(other, hamming_cutoff, bit_error_rate)
+		max_difference = len(self.segment_hashes)
+		if matches == 0:
+			return max_difference
 		max_distance = matches * len(self.segment_hashes[0])
-		return matches + (1 - (sum_distance / max_distance))
+		tie_breaker = 1 - (sum_distance / max_distance)
```

Comment:
Please check all " / ", this project is still supporting Python 2, which doesn't convert int/int divisions to floats automatically.

Reply:
Good point!
There's one in whash too, line 307. Should I fix that?

Reply:
Yes, change "255" to "255."

Reply:
Alright, I've fixed that too. But travis is taking ages (an hour!?) to actually start the tests. And running python 3.5 locally doesn't seem to show the same failure as Travis has in the last few runs